core/server/data/validation/index.js

```diff
@@ -42,6 +43,27 @@ validator.extend('isTimezone', function isTimezone(str) {
     return moment.tz.zone(str) ? true : false;
 });
 
+validator.extend('isEmptyOrWebsite', function isEmptyOrWebsite(str) {
```

Comment:
> Rather than strict validation that can break perfectly valid URLs I think we should rather store URLs in a safer format

Discussed in Slack - we sanitise output, not input. 

So this PR should only update to the newest version of validator @AileenCGN